tamr_toolbox/dataset/manage.py

```diff
@@ -0,0 +1,225 @@
+from typing import List, Optional, Union
+
+from tamr_unify_client import Client
+from tamr_unify_client.dataset.resource import Dataset
+from tamr_unify_client.attribute.resource import AttributeSpec
+from tamr_unify_client.attribute.type import AttributeType
+
+from tamr_toolbox.models.data_type import JsonDict
+
+
+def create(
+    *,
+    tamr: Client,
+    dataset_name: str,
+    primary_keys: Optional[List[str]] = None,
```

Comment:
To my knowledge no tamr projects can handle more than one primary key. To that end should we reinforce that this primary key is a single value and not a list.

Reply:
You can have source datasets that have more than one primary key. You are correct that you can't use them as input datasets to a project but you can join them on a UD. We do this on Avid